### PR TITLE
Fix to do dispatch agent if mapped scene is not active when launching autopilot

### PR DESCRIPTION
**Pull request by nowsprinting**
#### Situation

Launch autopilot while in play mode or running player build.
Multiple scenes are loaded, but the scene the agent is mapped to is not active (it is loaded).

#### Actual

Did not dispatch agent.

#### Expected

Dispatch agent mapped in the loaded (not active) scene.

### Fixes

Check mapping all loaded scenes if active scene is not mapped on launch.

### Priority

I want it to be released at the same time as #76 

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

File: Runtime/Autopilot.cs [hunk 1]
```diff
@@ -8,9 +8,11 @@
 using DeNA.Anjin.Loggers;
 using DeNA.Anjin.Settings;
 using DeNA.Anjin.Utilities;
+using UnityEditor;
```
Comment (bo40) [inline Runtime/Autopilot.cs:11]: [ASK]
Maybe `#if UNITY_EDITOR ... #endif` symbol guard is necessary?

Comment (nowsprinting) [reply Runtime/Autopilot.cs]: Thanks! fix in 95acd366

**Comment by nowsprinting** [comment]
I will rebase after revert 6352869

**Review by bo40 — APPROVED** [review]
LGTM

**Comment by bo40** [comment]
@nowsprinting 

Thanks for your contribution.

**Pull request merged**